### PR TITLE
1436790/1438462: Part 24: 20 files with MSDN link fixes.

### PR DESCRIPTION
**Pull request by DennisLee-DennisLee**
Contributes to PR #[2037](https://github.com/dotnet/docs/issues/2037) and applies to the following work items:
- [User Story 1436790: Content Update: update MSDN links for .NET content](https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/1436790)
- [Task 1438462: Update MSDN links for .NET content](https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/1438462)

### CODE REVIEW

**Comment by DennisLee-DennisLee** [comment]
Again this PR mostly contains changes to the same recurring paragraph.

File: docs/framework/winforms/controls/how-to-design-a-windows-forms-layout-that-responds-well-to-localization.md [hunk 1]
```diff
@@ -17,35 +17,35 @@ Creating forms that are ready to be localized greatly speeds development for int
...
   
- There is extensive support for this task in Visual Studio.  See also [Walkthrough: Creating a Layout That Adjusts Proportion for Localization](https://msdn.microsoft.com/library/7k9fa71y\(v=vs.110\)).  
+ There is extensive support for this task in Visual Studio.  See also [Walkthrough: Creating a Layout That Adjusts Proportion for Localization](https://docs.microsoft.com/previous-versions/visualstudio/visual-studio-2010/7k9fa71y(v=vs.100)).  
   
  [!code-csharp[System.Windows.Forms.TableLayoutPanel.LocalizableForm#1](../../../../samples/snippets/csharp/VS_Snippets_Winforms/System.Windows.Forms.TableLayoutPanel.LocalizableForm/CS/localizableform.cs#1)]
  [!code-vb[System.Windows.Forms.TableLayoutPanel.LocalizableForm#1](../../../../samples/snippets/visualbasic/VS_Snippets_Winforms/System.Windows.Forms.TableLayoutPanel.LocalizableForm/VB/localizableform.vb#1)]  
   
 1.  [How to: Align and Stretch a Control in a TableLayoutPanel Control](how-to-align-and-stretch-a-control-in-a-tablelayoutpanel-control.md)
```
Comment (rpetrusha) [inline docs/framework/winforms/controls/how-to-design-a-windows-forms-layout-that-responds-well-to-localization.md]:
```suggestion
## Additional resources
1.  [How to: Align and Stretch a Control in a TableLayoutPanel Control](how-to-align-and-stretch-a-control-in-a-tablelayoutpanel-control.md)  
```
A nit unrelated to your PR: it's unclear why the code is immediately followed by a list of articles, so I thought adding a header would be a good idea.

File: docs/framework/winforms/controls/how-to-display-an-insertion-mark-in-a-windows-forms-listview-control.md [hunk 2]
```diff
@@ -34,13 +34,13 @@ The insertion mark in the <xref:System.Windows.Forms.ListView> control shows use
...
+ For information about building this example from the command line for Visual Basic or Visual C#, see [Building from the Command Line](../../../visual-basic/reference/command-line-compiler/building-from-the-command-line.md) or [Command-line Building With csc.exe](../../../csharp/language-reference/compiler-options/command-line-building-with-csc-exe.md). You can also build this example in Visual Studio by pasting the code into a new project.  
   
 ## See also
 - <xref:System.Windows.Forms.ListView>
 - <xref:System.Windows.Forms.ListView.InsertionMark%2A?displayProperty=nameWithType>
 - <xref:System.Windows.Forms.ListViewInsertionMark>
 - [ListView Control](../../../../docs/framework/winforms/controls/listview-control-windows-forms.md)
 - [ListView Control Overview](../../../../docs/framework/winforms/controls/listview-control-overview-windows-forms.md)
-- [Windows XP Features and Windows Forms Controls](https://msdn.microsoft.com/library/bc7fab94-fce9-4bf1-a8ad-a5837c91c3c0)
+- [Windows XP Features and Windows Forms Controls](https://docs.microsoft.com/previous-versions/visualstudio/visual-studio-2010/06da8ztk(v=vs.100))
```
Comment (rpetrusha) [inline docs/framework/winforms/controls/how-to-display-an-insertion-mark-in-a-windows-forms-listview-control.md]: This item can be deleted.

File: docs/framework/winforms/controls/how-to-enable-tile-view-in-a-windows-forms-listview-control.md [hunk 3]
```diff
@@ -51,11 +51,11 @@ Tile view icons and text
...
+ For information about building this example from the command line for Visual Basic or Visual C#, see [Building from the Command Line](../../../visual-basic/reference/command-line-compiler/building-from-the-command-line.md) or [Command-line Building With csc.exe](../../../csharp/language-reference/compiler-options/command-line-building-with-csc-exe.md). You can also build this example in Visual Studio by pasting the code into a new project.  
   
 ## See also
 - <xref:System.Windows.Forms.ListView>
 - <xref:System.Windows.Forms.ListView.TileSize%2A>
 - [ListView Control](../../../../docs/framework/winforms/controls/listview-control-windows-forms.md)
 - [ListView Control Overview](../../../../docs/framework/winforms/controls/listview-control-overview-windows-forms.md)
-- [Windows XP Features and Windows Forms Controls](https://msdn.microsoft.com/library/bc7fab94-fce9-4bf1-a8ad-a5837c91c3c0)
+- [Windows XP Features and Windows Forms Controls](https://docs.microsoft.com/previous-versions/visualstudio/visual-studio-2010/06da8ztk(v=vs.100))
```
Comment (rpetrusha) [inline docs/framework/winforms/controls/how-to-enable-tile-view-in-a-windows-forms-listview-control.md]: This item can be deleted.

**Review by rpetrusha — CHANGES_REQUESTED** [review]
This looks good, @DennisLee-DennisLee I've left one suggested change, and thought two of the cross-references could be eliminated.

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thanks for making the additional changes, @DennisLee-DennisLee. I'll merge your PR now.

**Pull request merged**